hash/sega_beena_cart.xml

```diff
@@ -206,6 +456,7 @@ license:CC0
 		<description>Yes! PreCure 5 Asonde Oboeyou Hiragana!</description>
 		<year>2007</year>
 		<publisher>Bandai</publisher>
+		<info name="alt_title" value="Yes！プリキュア5 あそんでおぼえよう！ひらがな！"/>
```

Comment:
Any reason you used a halfwidth 5 in the Japanese text?

Reply:
Fixed, thanks!